tests/UI/campaigns/tools/linkchecker.js

```diff
@@ -133,8 +170,8 @@ async function crawlPage(browserPage, thisPageToCrawl) {
  */
 async function checkResponseStatus(url, status) {
   if (status.startsWith('4') || status.startsWith('5')) {
-    requestError = true;
-    requestTextError = `Request error : ${url} (${status}`;
+    responseError = true;
+    responseTextError = `Request error : ${url} (${status}`;
```

Comment:
Is it normal that parenthesis are not closed?

Reply:
should not be a parenthesis, nice catch

Reply:
sorry it was parenthesis :laughing: